runtime/vara/src/lib.rs

```diff
@@ -1021,12 +1021,16 @@ parameter_types! {
     pub Schedule: pallet_gear::Schedule<Runtime> = Default::default();
     pub BankAddress: AccountId = BANK_ADDRESS.into();
     pub const GasMultiplier: common::GasMultiplier<Balance, u64> = common::GasMultiplier::ValuePerGas(VALUE_PER_GAS);
+    pub SplitGas: Option<(Perbill, AccountId)> = None;
```

Comment:
please open todo to move this logic into pallet storage to be able to adjust the value by governance avoiding runtime upgrades